### PR TITLE
Add Comment And Article Models.

### PR DESCRIPTION
**Pull request by darkwlf**

### CODE REVIEW

File: account/models.py [hunk 1]
```diff
@@ -35,3 +35,17 @@ class Validation(models.Model):
     class Meta:
         db_table = 'pyfarsi_validations'
         ordering = ('-user__date_joined',)
+
+
+class Article(models.Model):
+    # TODO : Add Category Choices
+    title = models.CharField(max_length=90, default='')
```
Comment (parsa-rahbari-82) [inline account/models.py:42]: remove default

Comment (RezFD) [reply account/models.py]: blank false 
default = blank

File: account/models.py [hunk 2]
```diff
@@ -35,3 +35,26 @@ class Validation(models.Model):
...
+
+
+class Article(models.Model):
+    # TODO : Add Category Choices
+    title = models.CharField(max_length=90)
+    image = models.ImageField(upload_to='media')
+    slug = models.SlugField(max_length=4000)
+    publisher = models.ForeignKey(User, on_delete=models.CASCADE)
+    created = models.DateTimeField(auto_add_now=True)
+    STATUS_CHOICES = (
```
Comment (RezFD) [inline account/models.py:47]: Do not use tuple for choices

File: account/models.py [hunk 3]
```diff
@@ -35,3 +35,26 @@ class Validation(models.Model):
...
         db_table = 'pyfarsi_validations'
         ordering = ('-user__date_joined',)
+
+
+class Article(models.Model):
+    # TODO : Add Category Choices
+    title = models.CharField(max_length=90)
+    image = models.ImageField(upload_to='media')
+    slug = models.SlugField(max_length=4000)
+    publisher = models.ForeignKey(User, on_delete=models.CASCADE)
```
Comment (RezFD) [inline account/models.py:45]: change on delete to set null

File: account/models.py [hunk 4]
```diff
@@ -35,3 +35,26 @@ class Validation(models.Model):
     class Meta:
         db_table = 'pyfarsi_validations'
         ordering = ('-user__date_joined',)
+
+
+class Article(models.Model):
+    # TODO : Add Category Choices
+    title = models.CharField(max_length=90)
+    image = models.ImageField(upload_to='media')
+    slug = models.SlugField(max_length=4000)
```
Comment (RezFD) [inline account/models.py:44]: slug is unique

File: account/models.py [hunk 5]
```diff
@@ -35,3 +35,26 @@ class Validation(models.Model):
...
+    publisher = models.ForeignKey(User, on_delete=models.CASCADE)
+    created = models.DateTimeField(auto_add_now=True)
+    STATUS_CHOICES = (
+    ('d', 'Draft'),
+    ('p', 'Publish'),
+    )
+    status = models.CharField(max_length=1, choices=STATUS_CHOICES, default='p')
```
Comment (RezFD) [inline account/models.py:51]: default = draft

File: account/models.py [hunk 6]
```diff
@@ -35,3 +35,26 @@ class Validation(models.Model):
...
+
+
+class Comment(models.Model):
+    body = models.TextField(max_length=700)
+    user = models.ForeignKey(User, on_delete=models.CASCADE)
+    article = models.ForeignKey(Article, on_delete=models.CASCADE)
+    reply = models.ForeignKey('self', on_delete=models.CASCADE, null=True, blank=True, related_name="rcomment")
+    is_reply = models.BooleanField(defualt=False)
```
Comment (RezFD) [inline account/models.py:59]: remove is_reply

File: account/models.py [hunk 7]
```diff
@@ -35,3 +35,26 @@ class Validation(models.Model):
...
+    created = models.DateTimeField(auto_add_now=True)
+    STATUS_CHOICES = (
+    ('d', 'Draft'),
+    ('p', 'Publish'),
+    )
+    status = models.CharField(max_length=1, choices=STATUS_CHOICES, default='p')
+
+
+class Comment(models.Model):
+    body = models.TextField(max_length=700)
```
Comment (RezFD) [inline account/models.py:55]: change body name to content

**Comment by RealNitroZeus** [comment]
Maybe next time :smile:

**Pull request closed without merging**